app/gui/vite.config.ts

```diff
@@ -21,20 +21,26 @@ if (isDevMode) {
 // Used by vite middleware inside devtools plugin. Specifying this by an option doesn't work when `componentInspector` is false.
 process.env.LAUNCH_EDITOR ??= 'code'
 
+// These can be inlined, but are refactored out because `defineConfig` (understandably)
+// does not support async functions.
+const vueDevtoolsVitePlugin =
```

Comment:
I don't understand this comment. We passed awaited values to defineConfig, no async function.